knowledge-base/postgres-stored-functions-in-sqldatasource-wizard.md

```diff
@@ -22,15 +22,14 @@ res_type: kb
 ## Description
 In PostgreSQL prior to version 11 there are Stored functions. Stored Procedures have been introduced with version 11.
 
-1. When using ODBC provider for PostgreSQL with Stored functions, the Stored Procedure drop-down menu of the Report Designer's SqlDataSource wizard is populated, but the returned table columns will be listed as Stored Procedure arguments along with the real arguments, and the Stored function will not be correctly discovered.
-
+1. When using ODBC provider for PostgreSQL with Stored functions, the Stored Procedure drop-down menu of the Report Designer's SqlDataSource wizard is populated, but the returned table columns will be listed as Stored Procedure arguments along with the real arguments, and the Stored function will not be correctly discovered, resulting in an exception.
 
 2. When using NpgSql provider the Report Designer's SqlDataSource wizard does not list the PostgreSQL Stored functions in the Stored Procedures drop-down menu. The issue applies to PostgreSQL prior to version 11. 
```

Comment:
The wizard does not list the stored functions, because they are not provided in the schema information we request - maybe it's better to include it so we won't look lazy or negligent.

Reply:
corrected